tests/test_cli_client.cpp

```diff
@@ -66,8 +63,11 @@ struct Client : public Test
     mpt::StubCertProvider cert_provider;
     mpt::StubCertStore cert_store;
     mp::DaemonRpc stub_daemon{server_address, mp::RpcConnectionType::insecure, cert_provider, cert_store};
+    static std::stringstream trash_stream; // this may have contents (that we don't care about)
 };
 
+std::stringstream Client::trash_stream; // replace with inline in C++17
```

Comment:
Why not this: https://pastebin.ubuntu.com/p/BjsYZB6zWr/
Simpler, works now, no need for C++17 magic.

Reply:
> Simpler

For the writer only, and then only marginally.

> no need for C++17 magic

No "need" in either case.

> Why not this [...]

Moving it from its natural scope and making it a global arguably reduces clarity, so I don't think it is an improvement. Don't get me wrong, I don't think it would be bad either, this being tests at all. Almost indifferent perhaps, but not better IMO.

Reply:
I'm not gonna block on it